src/libcollections/bit.rs

```diff
@@ -1808,99 +1801,108 @@ impl hash::Hash for BitSet {
     }
 }
 
-/// An iterator for `BitSet`.
 #[derive(Clone)]
 #[stable(feature = "rust1", since = "1.0.0")]
-pub struct SetIter<'a> {
-    set: &'a BitSet,
-    next_idx: usize
+struct BlockIter<T> where
+    T: Iterator<Item=u32> {
```

Comment:
This vindicates my already held ideas about where clauses and brace style (newline after the where stuff).